modules/tracing/mule-tracer-common/src/main/java/org/mule/runtime/tracer/common/watcher/TracingConfigurationFileWatcher.java

```diff
@@ -6,11 +6,20 @@
  */
 package org.mule.runtime.tracer.common.watcher;
 
+import static java.nio.file.StandardWatchEventKinds.ENTRY_DELETE;
```

Comment:
order of static imports.
Usually we follow this convention:
static imports
       -> mule -> java -> other ones
import
       -> mule classes -> java -> other ones.